### PR TITLE
Task/284 redesign change password

### PR DESCRIPTION
**Pull request by TomasPetrik**
Update redesign change password screen to be matching with:

https://www.figma.com/file/JwEHS4hCfJvzBf5ZNXkg7H/Simplio-UI?node-id=659%3A35278

### CODE REVIEW

File: lib/view/screens/password_change_screen.dart [hunk 1]
```diff
@@ -3,144 +3,221 @@ import 'package:flutter_bloc/flutter_bloc.dart';
...
+                            color: SioColors.mentolGreen,
+                            size: 70,
+                          ),
+                          Positioned(
+                            left: 0,
+                            right: 0,
+                            bottom: 10,
+                            child: Icon(
+                              SioIcons.refresh,
+                              color: SioColors.highlight,
```
Comment (jvrastil) [inline lib/view/screens/password_change_screen.dart:71]: can you add some todo comment about replacing with correct icons?

**Review by jvrastil — CHANGES_REQUESTED** [review]

File: lib/view/screens/password_change_screen.dart (hunk 1 above)
Comment (TomasPetrik) [reply lib/view/screens/password_change_screen.dart:71]: Done.

File: lib/data/http/services/asset_service.dart [hunk 2]
```diff
@@ -41,7 +41,7 @@ class CryptoAssetResponse {
   final String regularFee;
   final String highFee;
   final String gasLimit;
-  final String feeUnit;
+  final String? feeUnit;
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/data/http/services/asset_service.dart]: Reverted back to be not nullable.

**Review by jvrastil — APPROVED** [review]
looks good now 👍

**Pull request merged**